src/fmeval/eval_algorithms/qa_accuracy.py

```diff
@@ -105,8 +107,11 @@ def _f1_score(model_output: str, target_output: str, *, normalize_text: bool = F
     :param model_output: The output of a model that we want to evaluate.
     :param target_output: The reference or the "ground truth" output.
     :param normalize_text: Normalize the text before computing f1.
+    :param strip_text: Strip the model_output and the target_output before computing the f1 score.
```

Comment:
Unclear how `normalize_text` and `strip_text` relate. A user might expect normalizing to include stripping (it doesn't). The functionality is not indicated by the name of the argument nor explained in the docstring. 

Easiest fix would be to update the docstring: 
- "We normalize the text following the QuAC protocol above." --> Delete
- ":param normalize_text: Normalize the text before computing f1." --> ":param normalize_text: Normalize the text before computing f1. We normalize the text following
    the QuAC protocol which includes removal of punctuation, stop words [TODO: anything else?]"
- ":param strip_text: Strip the model_output and the target_output before computing the f1 score." --> ":param strip_text: Strip the model_output and the target_output before computing the f1 score. I.e., remove trailing whitespaces and newlines. [TODO: anything else?]"

Reply:
Thanks Pola. I updated the docstring to differentiate between `normalize_text` and `strip_text`. Didn't list all the operations in front of `normalize_text` since the `_normalize_text_quac_protocol` function has that info. Same for whitespaces since the users can find the list of whitespace characters under `string.whitespace`.

Reply:
`_normalize_text_quac_protocol:` docstring says 
    "Given a text, normalize it using the SQUAD / QUAC protocol. That is remove punctuations, excess whitespaces [...]". _Whitespaces_ needs to be removed here. (In fact, this is probably where the confusion originates! This is why I thought we already remove whitespaces.)

Reply:
Discussed offline with Pola and changed "excess whitespaces" to "excess spaces" in the docstring of `_normalize_text_quac_protocol`.